### ISSUE TITLE
CLI call `generate <nblocks>` doesn't work on regtest.

### ISSUE THREAD

**Issue by leehankyeol** [digibyte/digibyte#114]
### Describe the issue

1. Download the latest (as of now) DigiByte, 6.14.2.
2. Configure as follows.
```
server=1
daemon=1
regtest=1
algo=scrypt # When not specified, "generate" call is blocked by multialgo validation. https://github.com/digibyte/digibyte/pull/73/commits/a4635b2816fab686abe15c6a1ad79528975152c9?diff=split
port=18000

rpcuser=testuser
rpcpassword=testpassword
rpcport=18001
```
3. Run DigiByte daemon and try to call `generate 101` CLI command via `digibyte-cli`.

### Expected behaviour

101 blocks should be generated.

### Actual behaviour

```
error code: -1
error message:
Division by zero
```

### What version of digibyte-core are you using?
6.14.2

### Machine specs:
- OS: CentOS 7.4

### Any extra information that might be useful in the debugging process.

At least `CreateNewBlock()` is logged.

```
2018-02-21 09:03:20 keypool reserve 5
2018-02-21 09:03:20 block height for reward is 1
2018-02-21 09:03:20 CreateNewBlock(): total size: 225 block weight: 900 txs: 0 fees: 0 sigops 400
2018-02-21 09:03:20 keypool return 5
```

**Comment by skawhdgjs** [comment]
Regtest netparam have to be filled with consenesus values

**Comment by digicontributer** [comment]
Fixed in 6.16.2
